Tests/AppTests/AppTestCase.swift

```diff
@@ -40,20 +40,14 @@ class AppTestCase: XCTestCase {
 
 
 extension AppTestCase {
-    func renderSQL(_ builder: SQLSelectBuilder?, resolveBinds: Bool = false) -> String {
-        renderSQL(builder?.query, resolveBinds: resolveBinds)
+    func renderSQL(_ builder: SQLSelectBuilder?) -> String {
+        renderSQL(builder?.query)
     }
```

Comment:
Removing the `resolveBinds:` parameter forces all tests to assert both the sql with the substitution parameters (`$1`, `$2`, ...) and the binds separately. A bit more work but the big advantage is that we can see and ensure that binds are always used and filled in as we expect.

Reply:
Yea this is a great change. I can see how the original misunderstanding happened!